v2/cf-nodejs-logging-support/package.json

```diff
@@ -35,15 +35,15 @@
     "@types/express": "^4.17.13",
     "@types/node": "^17.0.45",
     "@types/restify": "^8.5.4",
-    "@typescript-eslint/eslint-plugin": "^4.0.1",
-    "@typescript-eslint/parser": "^4.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^3.10.1",
     "chai": "^4.3.6",
-    "eslint": "^7.8.0",
+    "eslint": "^8.20.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-functional": "^3.0.2",
     "eslint-plugin-import": "^2.22.0",
     "import-fresh": "^3.3.0",
-    "mocha": "^9.2.2",
+    "mocha": "^6.2.3",
```

Comment:
Why is this downgrade required?

Reply:
Must have been downgraded by mistake. I turned it back to version 9.2.2.